code/modules/busy_space/organizations.dm

```diff
@@ -0,0 +1,461 @@
+	destination_names = list(
+	"a outpost on Jargon 4"
+	)
+
+/datum/lore/organization/tsc/zeng_hu
+	name = "Zeng-Hu pharmaceuticals"
+	short_name = "Zeng-Hu"
+	acronym = "ZH"
+	desc = "Zeng-Hu is an old TSC, based in the Sol system. Until the discovery of Plasma, Zeng-Hu maintained a stranglehold \
+	on the market for medications, and many household names are patentted by Zeng-Hu-- Bicaridyne, Dylovene, Tricordrizine, \
+	and Dexalin all came from a Zeng-Hu medical laboratory. Zeng-Hu's fortunes have been in decline as Nanotrasen's near monopoly \
+	on plasma research cuts into their R&D and Vey-Med's superior medical equipment effectively decimated their own equipment \
+	interests. The three-way rivalry between these companies for dominance in the medical field is well-known and a matter of \
+	constant economic speculation."
+	history = ""
+	work = "pharmaceuticals company"
+	headquarters = ""
+	motto = ""
+
+	ship_prefixes = list("ZTV" = "transportation", "ZMV" = "medical resupply")
+	destination_names = list()
+
+/datum/lore/organization/tsc/ward_takahashi
+	name = "Ward-Takahashi General Manufacturing Conglomerate"
+	short_name = "Ward-Takahashi"
+	acronym = "WT"
+	desc = "Ward-Takahashi focuses on the sale of small consumer electronics, with its computers, communicators, \
+	and even mid-class automobiles a fixture of many households. Less famously, Ward-Takahashi also supplies most \
+	of the AI cores on which vital control systems are mounted, and it is this branch of their industry that has \
+	led to their tertiary interest in the development and sale of high-grade AI systems. Ward-Takahashi's economies \
+	of scale frequently steal market share from Nanotrasen's high-price products, leading to a bitter rivalry in the \
+	consumer electronics market."
+	history = ""
+	work = "electronics manufacturer"
+	headquarters = ""
+	motto = ""
+
+	ship_prefixes = list("WTV" = "freight")
+	destination_names = list(
+	"a station near Kelune"
+	)
+
+/datum/lore/organization/tsc/bishop
+	name = "Bishop Cybernetics"
+	short_name = "Bishop"
+	acronym = "BC"
+	desc = "Bishop's focus is on high-class, stylish cybernetics. A favorite among transhumanists (and a b'te noire for \
+	bioconservatives), Bishop manufactures not only prostheses but also brain augmentation, synthetic organ replacements, \
+	and odds and ends like implanted wrist-watches. Their business model tends towards smaller, boutique operations, giving \
+	it a reputation for high price and luxury, with Bishop cyberware often rivalling Vey-Med's for cost. Bishop's reputation \
+	for catering towards the interests of human augmentation enthusiasts instead of positronics have earned it ire from the \
+	Positronic Rights Group and puts it in ideological (but not economic) competition with Morpheus Cyberkinetics."
+	history = ""
+	work = "cybernetics and augmentation manufacturer"
+	headquarters = ""
+	motto = ""
+
+	ship_prefixes = list("BTV" = "transportation")
+	destination_names = list()
+
```

Comment:
/datum/lore/organization/tsc/shellguard
	name = "Shellguard Munitions"
	short_name = "Shellguard"
	acronym = "SM"
	desc = "The brainchild of a colonial war veteran, Shellguard Munitions is an arms manufacturer and private military contractor specializing in anti-synthetic weapon systems and platforms. Initially a smaller private military force only serving frontier colonies, Shellguard Munitions has become a household name due to its involvement in resolving the Haverick AI crisis in 2552. Using its recently earned fame, the company has made a successful foray into the market of robotics and is highly regarded for the toughness and reliability of their hardware. Despite being frequently contracted by the Trans-Solar Federation, Shellguard Munitions is known to sell their services to the highest corporate bidder."
	history = ""
	work = "anti-synthetic arms manufacturer and PMC"
	headquarters = "Colony of Haverick"
	motto = ""

	ship_prefixes = list("SMT" = "transportation")
	destination_names = list()